memtable/write_buffer_manager_test.cc

```diff
@@ -47,7 +47,33 @@ TEST_F(WriteBufferManagerTest, ShouldFlush) {
   ASSERT_TRUE(wbf->ShouldFlush());
 
   wbf->FreeMem(7 * 1024 * 1024);
-  // 9MB total, 8MB mutable.
+  // 8MB total, 8MB mutable.
+  ASSERT_FALSE(wbf->ShouldFlush());
+
+  // change size: 8M limit, 7M mutable limit
```

Comment:
Can you add the test if buffer size is decreased as well.

Reply:
@akankshamahajan15 Original buffer size was 10M and my added test lowers it to 8M.  I am confused as to what to write.

Reply:
My bad. Its fine.